packages/app-root/src/app/about/publications/page.js

```diff
@@ -0,0 +1,51 @@
+import { Publications } from '@zooniverse/content'
+
+import {
+  buildResponse,
+  createProjectAvatarsMap,
+  getProjectAvatars,
+  getPublicationsData,
+  getUniqueProjectIds
+} from './processPublicationData.js'
+import client from '../../../utils/contentfulClient.js'
+import mockResponse from './response.mock.json'
+
+export const metadata = {
+  title: 'Our Team',
+  description: 'The people who make the Zooniverse'
+}
+
+// This route is static, so the output of the request will be cached and revalidated as part of the route segment.
+async function createPublicationsResponse() {
+  if (client) {
```

Comment:
It's unlikely there'd be an issue, but we could wrap the if statement with a try-catch and return the mockResponse in the catch as well. Not blocking, for consideration.

Reply:
True!